cli_tools/common/disk/disk.proto

```diff
@@ -0,0 +1,27 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+syntax = "proto3";
+
+option go_package = ".;disk";
+
+enum PartitionScheme {
+  NONE = 0;
```

Comment:
I will use partition type instead of partition style for the UEFI feature. The output will be HasEFI=true or fals.e

Reply:
sgtm; removed.

Also, I removed the protobufs, since they felt awkward given that we're not actually using them for anything yet. I replaced them with a normal Go struct; as a side effect, it'll be much easier for you to update!